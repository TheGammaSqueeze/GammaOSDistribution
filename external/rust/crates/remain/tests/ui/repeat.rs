enum E {
    Aaa(u8),
    Bbb,
}

#[remain::check]
fn main() {
    #[sorted]
    match E::Bbb {
        E::Aaa(0) => {}
        E::Bbb => {}
        E::Aaa(_) => {}
    }
}
