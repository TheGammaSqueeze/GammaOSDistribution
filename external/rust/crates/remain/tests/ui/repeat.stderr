error: E::Aaa should sort before E::Bbb
  --> $DIR/repeat.rs:12:9
   |
12 |         E::Aaa(_) => {}
   |         ^^^^^^
